lib/common/client/base.rb

```diff
@@ -90,7 +90,7 @@ def sanitize_headers!(method, path, params, headers)
         end
       end
 
-      def get(path, params, headers = base_headers)
```

Comment:
I noticed that `base_headers` is not defined anywhere, and thus would cause an exception if it was ever called.

Reply:
i wouldn't change this in this pr, how about making another pr where it's removed from every method and tag kam in it?

Reply:
Good idea.  Will do.